src/index.tsx

```diff
@@ -24,6 +25,7 @@ import * as config from 'config'
 import { getApi, setApi } from '@/api/Api'
 import MapActionReceiver from '@/stores/MapActionReceiver'
 import { createMap, getMap, setMap } from '@/map/map'
+import MapFeatureStore from '@/stores/MapFeatureStore'
```

Comment:
I did not know where else to put the coordinate and contents of the hover popup, so I added a new store that we can maybe use for other similar things as well...